### PR TITLE
Added C353991 - Warning message appears when creating order from instance record and select not existing order

### PR DESCRIPTION
**Pull request by KazachenkaTatsiana**
https://foliotest.testrail.io/index.php?/cases/view/353991
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5722/console
image

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+const item = {
+  instanceName: `inventory_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+const wrongMessage = 'PO number does not exist';
+const wrongPO = `${randomFourDigitNumber()}`;
+
+describe('Inventory interaction', () => {
```
Comment (manowarqqq) [inline cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js:14]: describe('Orders', () => {
describe('Inventory interaction', () => {

File: cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  before('Create test data', () => {
+    cy.getAdminToken();
+    InventoryInstances.createInstanceViaApi(item.instanceName, item.itemBarcode);
```
Comment (manowarqqq) [inline cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js:17]: Need to create user with permissions
Existing user with follow permissions is logged in:
-Inventory: Create order from instance
-Inventory: View, create, edit holdings
-Inventory: View, create, edit instances
-Inventory: View, create, edit items
-Orders: Can view Orders and Order lines

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 3]
```diff
@@ -844,4 +848,16 @@ export default {
       Section({ id: 'acc02' }).find(KeyValue('Resource title')).has({ value: instanceData.title }),
     ]);
   },
+  clickNewOrder() {
+    cy.do([instanceDetailsSection.find(actionsButton).click(), newOrderButton.click()]);
+  },
+  varifyModalDialogExists() {
```
Comment (manowarqqq) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:854]: No need this method
There is fragment/inventory/modals/newOrderModal
use NewOrderModal.waitLoading();

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 4]
```diff
@@ -844,4 +848,16 @@ export default {
       Section({ id: 'acc02' }).find(KeyValue('Resource title')).has({ value: instanceData.title }),
     ]);
   },
+  clickNewOrder() {
```
Comment (manowarqqq) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:851]: already exists in 
https://github.com/folio-org/stripes-testing/pull/2504
InventoryInstance.newOrder();

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 5]
```diff
@@ -844,4 +848,16 @@ export default {
...
+    cy.do([instanceDetailsSection.find(actionsButton).click(), newOrderButton.click()]);
+  },
+  varifyModalDialogExists() {
+    cy.expect(modalDialog.exists());
+  },
+  fullAndCreatePONumber: (poNumber) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:857]: already exists in
https://github.com/folio-org/stripes-testing/pull/2504
or
https://github.com/folio-org/stripes-testing/pull/2526

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 6]
```diff
@@ -844,4 +848,16 @@ export default {
...
+    cy.do([TextField({ name: 'poNumber' }).fillIn(poNumber), createButton.click()]);
+  },
+  varifyTextMessageExists(text) {
```
Comment (manowarqqq) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:860]: move to fragment/inventory/modals/newOrderModal

File: cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    () => {
+      InventorySearchAndFilter.switchToInstance();
+      InventorySearchAndFilter.byKeywords(item.instanceName);
+      InventoryInstance.checkInstanceTitle(item.instanceName);
+
+      InventorySearchAndFilter.clickNewOrder();
+      InventorySearchAndFilter.varifyModalDialogExists();
+
+      InventorySearchAndFilter.fullAndCreatePONumber(wrongPO);
+      InventorySearchAndFilter.varifyTextMessageExists(wrongMessage);
```
Comment (manowarqqq) [inline cypress/e2e/inventory/warning-message-when-creating-non-existent-order.cy.js:41]: Add expected result for
Create order" popup is displayed
NewOrderModal.waitLoading();

**Pull request merged**